osu.Game.Tests/Visual/UserInterface/TestSceneOverlayHeader.cs

```diff
@@ -36,11 +36,11 @@ public TestSceneOverlayHeader()
                 }
             });
 
-            addHeader("Orange OverlayHeader (no background)", new TestNoBackgroundHeader(), OverlayColourScheme.Orange);
-            addHeader("Blue OverlayHeader", new TestNoControlHeader(), OverlayColourScheme.Blue);
+            addHeader("Orange OverlayHeader (no background, 100 padding)", new TestNoBackgroundHeader(), OverlayColourScheme.Orange);
+            addHeader("Blue OverlayHeader (default 70 padding)", new TestNoControlHeader(), OverlayColourScheme.Blue);
```

Comment:
this is actually incorrect. the default is 50, as far as I can tell, and the draw visualiser seems to indicate the same.

Reply:
oops, forgot to rename this after latest commit